### PR TITLE
samtools view forgotten commits

### PR DESCRIPTION
**Pull request by bernt-matthias**
I accidentally commited two important commits intended for https://github.com/galaxyproject/tools-iuc/pull/1963
to my master branch. 

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

FOR REVIEWER:
* [x] .shed.yml file ok
    - [x] Toolshed user `iuc` has access to associated toolshed repo(s)
* [x] Indentation is correct (4 spaces)
* [x] Tool version/build ok
* [x] `<command/>`
  - [x] Text parameters, input and output files `'single quoted'`
  - [x] Use of `<![CDATA[ ... ]]>` tags
  - [x] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [x] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [x] Tests
  - [x] Parameters are reasonably covered
  - [x] Test files are appropriate
* [x] Help
  - [x] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [x] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

### CODE REVIEW

**Comment by bernt-matthias** [comment]
ping @mvdbeek

File: tool_collections/samtools/samtools_view/samtools_view.xml [hunk 1]
```diff
@@ -83,10 +83,13 @@
...
             -T '$reffa'
             -t '$reffai'
-        #else:
-            --output-fmt-option no_ref
         #end if
 
+        ##currently reference based CRAM is disabled (see https://github.com/galaxyproject/tools-iuc/pull/1963)
+        #if outtype=='cram':
+            --output-fmt-option no_ref
+        #endif
```
[the remark this answers is not in the document]
Comment (bernt-matthias) [reply tool_collections/samtools/samtools_view/samtools_view.xml]: done

File: tool_collections/samtools/samtools_view/samtools_view.xml [hunk 2]
```diff
@@ -83,7 +83,10 @@
         #if $reffa!=None:
             -T '$reffa'
             -t '$reffai'
-        #else:
+        #end if
+
+        ##currently reference based CRAM is disabled (see https://github.com/galaxyproject/tools-iuc/pull/1963)
+        #if outtype=='cram':
```
Comment (mvdbeek) [inline tool_collections/samtools/samtools_view/samtools_view.xml]: Missed this one, it should be `$outtype`

File: tool_collections/samtools/samtools_view/samtools_view.xml [hunk 3]
```diff
@@ -191,8 +194,8 @@
             <param name="addref_select" type="select" label="Reference data">
 		    <help>Reference data as fasta(.gz). Required for SAM input without @SQ headers and useful/required for writing CRAM output (see help).</help>
                 <option value="no" selected="True">No, see help (-output-fmt-option no_ref)</option>
-		<option value="history">Use a genome/index from the history (-t/-T)</option>
-		<option value="cached">Use a built-in genome (-t/-T)</option>
+		            <option value="history">Use a genome/index from the history (-t/-T)</option>
```
Comment (mvdbeek) [inline tool_collections/samtools/samtools_view/samtools_view.xml]: There are 2 leading tabs here, they should be spaces

File: tool_collections/samtools/samtools_view/samtools_view.xml (hunk 2 above)
Comment (bernt-matthias) [reply tool_collections/samtools/samtools_view/samtools_view.xml]: ups. sorry

File: tool_collections/samtools/samtools_view/samtools_view.xml (hunk 3 above)
Comment (bernt-matthias) [reply tool_collections/samtools/samtools_view/samtools_view.xml]: fixed

**Comment by mvdbeek** [comment]
awesome, thanks @bernt-matthias !

**Pull request merged**